core/src/main/scala/org/http4s/QValue.scala

```diff
@@ -13,10 +15,7 @@ import scala.reflect.macros.whitebox.Context
   * @param thousandths between 0 (for q=0) and 1000 (for q=1)
   * @see [[http://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.9 RFC 2616, Section 3.9]]
   */
-final class QValue private[QValue] (val thousandths: Int)
-    extends AnyVal
-    with Ordered[QValue]
-    with Renderable {
+final class QValue private[QValue] (val thousandths: Int) extends AnyVal with Ordered[QValue] {
```

Comment:
Does `private[QValue]` mean anything different than `private`?  This is not new, but it looks weird.

Reply:
I guess plain `private` will do just fine.

Reply:
Note that i removed `with Renderable` as it doesn't seem to be used as `Renderable` but you may have a different opinion